core/themes/basis/css/component/node.css

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * Styling for node content.
+ */
+
+.node {
+  position: relative;
+}
+
+/* Display "Draft" watermark on unpublished content. */
+/* See basis_preprocess_node(). */
+.node[data-status]:before {
+  content: attr(data-status);
```

Comment:
- shouldn't it be `.node.unpublished[data-status]:before`, to prevent displaying this data attribute on other case than unpublished ?
- CSS3 syntax is `::before`, not `:before`. See https://developer.mozilla.org/en/docs/Web/CSS/::before